api/DOMRect.json

```diff
@@ -272,6 +368,102 @@
             "deprecated": false
           }
         }
+      },
+      "x": {
```

Comment:
This is `x` which wasn't part of `ClientRect`. Can you make this and `y` match the parent feature instead?

Reply:
I'll suggest the specific edits.